.idea/allianceauth.iml

```diff
@@ -6,21 +6,16 @@
         <option name="rootFolder" value="$MODULE_DIR$" />
         <option name="settingsModule" value="settings.py" />
         <option name="manageScript" value="manage.py" />
+        <option name="environment" value="&lt;map/&gt;" />
       </configuration>
     </facet>
   </component>
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 2.7.6 (/usr/bin/python2.7)" jdkType="Python SDK" />

```

Comment:
I looked it up, the .idea folder is pycharms own folder with info about the project. The .iml file defines the project structure. 

The row
jdkName="Python 2.7.6 (/usr/bin/python2.7)"
ascertains that the 2.7 interpreter is chosen for the code validation.